src/main/java/com/microsoft/aad/msal4j/AuthenticationResultSupplier.java

```diff
@@ -132,6 +128,22 @@ private void logResult(AuthenticationResult result, HttpHeaders headers) {
         }
     }
 
+     private void logException(Exception ex) {
+
+         String logMessage = LogHelper.createMessage("Execution of " + this.getClass()
+                 + " failed.", msalRequest.headers().getHeaderCorrelationIdValue());
+
+         if (ex instanceof MsalClientException) {
```

Comment:
probably we need to log not MsalClientExceptions also, looks like clientApplication.log.error(logMessage, ex); line should be moved from if closure

Reply:
Good catch, updated.